modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/store/FilePageStoreManagerTest.java

```diff
@@ -159,6 +157,14 @@ void testInitialize() throws Exception {
         }
     }
 
+    private static void ensure(GroupPartitionPageStore<FilePageStore> filePageStore) {
+        try {
+            filePageStore.pageStore().ensure();
+        } catch (IgniteInternalCheckedException e) {
+            throw new RuntimeException(e);
```

Comment:
Hm? Maybe we shouldn't do this, right? There are two options:
- proper exception class
- don't use "forEach", use good old loop
I like the loop much more